R/post_processing_functions.R

```diff
@@ -12,33 +14,31 @@ smc_processing<-function(output){
   cnames   <- c(paste(cletters, cindexes, sep=" "))
   colnames(df) = cnames
 
-  new_df = gather(df, key = "item", value = "value")
+  new_df = tidyr::gather(df, key = "item", value = "value")

```

Comment:
Please use `<-` for assignment. I know the style isn't very consistent already, but I suggest we try the best we can :-)

Reply:
Not sure I did this commenting correctly.

Reply:
Sure thing! I was overlooking style formatting since most things will eventually be rewritten in C++ anyway (so using `<-` would be sort of a step back), but on second thought I think all these post-processing functions will stay in R, so it makes sense to tidy them up soon. Will comply!